src/actions/guides/database/migrations.cr

```diff
@@ -164,6 +164,7 @@ class Guides::Database::Migrations < GuideAction
       add login_count : Int32, default: 0
       add tags : Array(String)
       add preferences : JSON::Any
+      add key : String, index: true
```

Comment:
This isn't really related, but I wanted to sneak this in anyway to give an example that this is possible.